README.md

```diff
@@ -2,6 +2,67 @@
 
 This repository provides a GitHub Action for running the [Kani Rust Verifier](https://github.com/model-checking/kani) in CI.
 
+## GitHub Action Parameters
+
+The following parameters can be used to configure and customize the behavior of this GitHub Action:
+
+NOTE: All the fields provided are optional and have default behaviors when not specified.
+
+`kani-version`
+
+- **Description**: Specifies the Kani version number to use.
+- **Default**: 'latest'
+- **Example-pinned-version**: `'0.33.0'`
+- **Usage**: You can provide a specific version of Kani to use. [Cargo's version specific format is expected for specific versions](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html).
+If omitted, the latest version of `Kani` hosted on [`Kani's crates.io page`](https://crates.io/crates/kani-verifier) will be installed and used.
```

Comment:
Well done! Although I wonder if we should reduce the space for each parameters by putting all them into a list. For example, taking some inspiration from [this CodeGuru action](https://github.com/marketplace/actions/codeguru-security), we could have this option be:

```
 * `kani-version` (optional, default: `latest`): The Kani version to use. You can provide [...]
```

This would make the documentation feel much lighter.

Reply:
I've left it as it is for now, I think detail can be good, at least in the beginning. We can always adjust it later on.